.github/ISSUE_TEMPLATE/new-reference-doc.md

```diff
@@ -1,42 +1,32 @@
----
-name: "[MAINTAINERS] Add new reference document"
-about: FOR MAINTAINERS ONLY - Use this template to create an issue to add a new reference
-  document
-title: "[<LANG>] Add new reference document: <DOC>"
-labels: status/help-wanted, type/new-reference
-assignees: ''
-
 ---
 
-[Use this issue template to describe how to add a new reference doc. Only suggest adding a new reference doc when no appropriate online resource can be found. Please fill in the issue template, and remove any sections wrapped in square brackets (such as this section!).]
+name: "[MAINTAINERS] Add new reference document"
+about:
+FOR MAINTAINERS ONLY - Use this template to create an issue to add a new reference
+document
+title: "[Docs] Add new reference document: <DOC>"
+labels: status/help-wanted, type/new-reference-doc
+assignees: ""
+---[Use this issue template to describe how to add a new reference doc. Only suggest adding a new reference doc when no existing reference document has been written. Please fill in the issue template, and remove any sections wrapped in square brackets (such as this section!).]
```

Comment:
```suggestion
---

[Use this issue template to describe how to add a new reference doc. Only suggest adding a new reference doc when no existing reference document has been written. Please fill in the issue template, and remove any sections wrapped in square brackets (such as this section!).]
```

Not sure if Markdown frontmatter will work if there's text in the same line as the closing `---`.